src/stores/ui.ts

```diff
@@ -28,13 +34,10 @@ export interface UiState {
    * whether the request user agent is mobile or not.
    */
   isMobileUa: boolean
-}
-
-const cookieOptions: CookieSerializeOptions = {
-  path: '/',
-  sameSite: 'strict',
-  maxAge: 60 * 60 * 24 * 60, // 60 days
-  secure: isProd,
+  /**
+   * array of banner ids that were dismissed
+   */
```

Comment:
```suggestion
  /**
   * array of banner ids that were dismissed
   */
```

Not a necessary comment, imo.